js/components/ReportSummary.vue

```diff
@@ -3,12 +3,17 @@
     <div class="panel-body">
       <div class="container">
         <h3>{{ totalRecords }} - {{ title }}</h3>
+
```

Comment:
I've been trying to avoid logic in my templates. I'd probably create an `isItemized` computed property and use that to trigger the `v-if` instead. This also appears to lack a key for the `v-for`.